sdk/core/Azure.Core/tests/RequestContextTests.cs

```diff
@@ -284,6 +284,79 @@ public async Task CanSuppressTracingAsError()
             CollectionAssert.Contains(activity.Tags, new KeyValuePair<string, string>("otel.status_code", "UNSET"));
         }
 
+        [Test]
+        public async Task PerRequestClassifierTakesPrecedenceOverRetryClassifier()
+        {
+            // pipeline setup
+            var response = new MockResponse(429);
+            var mockTransport = new MockTransport(response);
+            var retryResponseClassifier = new ResponseClassifier();
+            var clientOptions = new TestOptions()
+            {
+                Transport = mockTransport,
+                RetryPolicy = new RetryPolicy(new RetryPolicyOptions { ResponseClassifier = retryResponseClassifier })
+            };
+            var pipeline = HttpPipelineBuilder.Build(clientOptions);
```

Comment:
We should initialize the pipeline ResposeClassifier here too, just to prove it can't override the others.

Reply:
We call into the same code path whether passing the classifier or not so I don't think this adds coverage.